### PR TITLE
Improve CodeExamples

### PR DESCRIPTION
**Pull request by sausman**
- Support single, plaintext example
- Add copy button
- Add `useCopy` hook

### CODE REVIEW

File: components/util/useCopy.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react";
```
Comment (sausman) [inline components/util/useCopy.tsx]: Where do you guys think we should put custom hooks? Not sure this is the best/most obvious place.

Comment (Finholt) [reply components/util/useCopy.tsx]: I agree that the util folder probably isn't quite the right place. In the demo app, we had them on them in a hooks folder on the same level as the components folder, but I could also see putting them in a hooks folder inside the components folder

File: components/partial/code/CodeExamples.tsx [hunk 2]
```diff
@@ -144,19 +105,31 @@ export default function CodeExamples({
...
+        {onlyOneExample ? (
+          <BarText>{activeLanguage}</BarText>
+        ) : (
+          <Select
+            autoWidth
+            variant="code"
+            options={options}
+            selectedValue={options.find((o) => o.value === activeLanguage)}
+            setSelectedValue={({ value }) => setSelectedLanguage(value)}
+          />
```
Comment (Finholt) [inline components/partial/code/CodeExamples.tsx:124]: I noticed that one of the options is `bash`, but we have it as `raw` in the dropdown that we have in the top nav. Can we switch one to the other so it's more consistent?

Comment (sausman) [reply components/partial/code/CodeExamples.tsx:124]: Yeah nice catch,

I updated the value in `LANGUAGE_OPTIONS` for now to be `bash`, but we may want to update the .mdx files to use ```raw. cc @spencerhunter

File: components/util/useCopy.tsx (hunk 1 above)
Comment (sausman) [reply components/util/useCopy.tsx]: Ended up moving to `/hooks`

**Review by Finholt — APPROVED** [review]

**Pull request merged**